agent/wpthook/test_state.cc

```diff
@@ -319,7 +325,14 @@ bool TestState::IsDone() {
   if (_active) {
     bool is_page_done = false;
     CString done_reason;
-    if (test_ms >= _test._minimum_duration) {
+    DWORD elapsed_timeout_ms = ElapsedMsFromTimeoutStart(now);

```

Comment:
Don't really need a whole new method for this, just call ElapsedMs directly with _timeout_start_time